tools/release-note.md

```diff
@@ -1,11 +1,21 @@
-This is release `${CIRCLE_TAG}` of Loki. 
+This is release `${CIRCLE_TAG}` of Loki.
 
 ### Notable changes:
 :warning: **ADD RELEASE NOTES HERE** :warning:
 
 ### Installation:
 The components of Loki are currently distributed in plain binary form and as Docker container images. Choose what fits your use-case best.
 
+#### Binary:
```

Comment:
I would remove this part. Downloading directly to /usr/local/bin/loki.zip looks like a bad idea, same as blindly unziping it there. (Not to mention that one needs permissions for writing to /usr/local/bin, ZIP includes "dist/loki" file, not just "loki")

I think that generic advise to download from <here> and don't forget to set +x permissions would do.